actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -190,10 +224,24 @@ def plugin_types(*types)
       end
     end
 
+    # Specify the {report-uri}(https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri)
+    # directive. Violation reports will be sent to the specified URI:
```

Comment:
"Enable" maybe?  

```suggestion
    # Enable the {report-uri}(https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri)
    # directive. Violation reports will be sent to the specified URI:
```

Or "Configure"?

```suggestion
    # Configure the {report-uri}(https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri)
    # directive. Violation reports will be sent to the specified URI:
```

Reply:
I think I prefer "Enable" because without using this method it's disabled.